src/NHibernate.Config.Templates/Oracle-Managed.cfg.xml

```diff
@@ -9,10 +9,10 @@ for your own use before compile tests in VisualStudio.
 	<session-factory name="NHibernate.Test">
 		<property name="connection.driver_class">NHibernate.Driver.OracleManagedDataClientDriver</property>
 		<property name="connection.connection_string">
-			User ID=nhibernate;Password=nhibernate;Data Source=(DESCRIPTION = (ADDRESS = (PROTOCOL = TCP)(HOST = localhost)(PORT = 1521)) (CONNECT_DATA = (SERVER = DEDICATED) (SERVICE_NAME = XE)))
+			User ID=nhibernate;Password=nhibernate;Data Source=(DESCRIPTION = (ADDRESS = (PROTOCOL = TCP)(HOST = localhost)(PORT = 1521)) (CONNECT_DATA = (SERVER = DEDICATED) (SERVICE_NAME = ORCL)))
```

Comment:
`ORCL` seems to be the default service name for 12c. `XE` is very likely the default name when using Express edition, which currently can at best be a 11g. Since instructions are now about installing a 12c, better switch to its default name. (But on the teamcity server, we are still using the `XE` name it seems.)